src/nvim/generators/gen_char_blob.lua

```diff
@@ -28,6 +28,8 @@ for argi = 2, #arg, 2 do
 
   target:write(('static const uint8_t %s[] = {\n'):format(varname))
 
+  local compiled = assert(io.popen(os.getenv("LUAC_PRG"):format(source_file)):read("*a"))
```

Comment:
`gen_char_blob` could be used for other things than lua source potentially. perhaps add an `arg` to toggle compilation?